### PR TITLE
README updated

### PR DESCRIPTION
**Pull request by sarastrasner**
- [X] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [X] README for this challenge is complete
     - [X] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [X] Picture of whiteboard

### CODE REVIEW

File: javascript/stacksAndQueues/fifoAnimalShelter/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Create a class called AnimalShelter which holds only dogs and cats. The shelter operates using a first-in, first-out approach.
+- Implement the following methods:
+  - enqueue(animal): adds animal to the shelter. animal can be either a dog or a cat object.
+  - dequeue(pref): returns either a dog or a cat. If pref is not "dog" or "cat" then return null.
+
+## Approach & Efficiency
+My team and I worked on the whiteboard for an hour. Big O for enqueue is O(1) and 0(n) for dequeue.
+
+## Solution
+![FiFo Animal Shelter](fifoAnimalShelter.png)
```
Comment (adrianhuebner) [inline javascript/stacksAndQueues/fifoAnimalShelter/README.md:14]: Off to a great start! Please take your time with this! I will come back and regrade this once you have worked out code and tests!
